app/src/androidTest/java/com/example/nsc_events/ForgotPasswordTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.nsc_events
```

Comment:
It's great to see that you are adding tests.

Reply:
Yes, thanks @taylorpapke was looking it over now. I like how it automates entries into the text input. Pretty Neat!